### ISSUE TITLE
undefined method `root' for #<EmberCli::App:0x007f80f21ba290>

### ISSUE THREAD

**Issue by thejchap** [thoughtbot/ember-cli-rails#295]
Happens on `rails g ember-cli:heroku`. Still investigating, just wanted to open up in case anyone else has run into this

Version 0.5.4
Ruby 2.2.2
Rails 4.2.1

```
/opt/rubies/2.2.2/lib/ruby/gems/2.2.0/gems/ember-cli-rails-0.5.4/lib/generators/ember-cli/heroku/heroku_generator.rb:32:in `block in app_paths': undefined method `root' for #<EmberCli::App:0x007f80f21ba290> (NoMethodError)
	from /opt/rubies/2.2.2/lib/ruby/gems/2.2.0/gems/ember-cli-rails-0.5.4/lib/generators/ember-cli/heroku/heroku_generator.rb:31:in `map'
	from /opt/rubies/2.2.2/lib/ruby/gems/2.2.0/gems/ember-cli-rails-0.5.4/lib/generators/ember-cli/heroku/heroku_generator.rb:31:in `app_paths'
	from (erb):9:in `template'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/2.2.0/erb.rb:863:in `eval'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/2.2.0/erb.rb:863:in `result'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions/file_manipulation.rb:116:in `block in template'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions/create_file.rb:53:in `call'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions/create_file.rb:53:in `render'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions/create_file.rb:46:in `identical?'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions/create_file.rb:72:in `on_conflict_behavior'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions/empty_directory.rb:113:in `invoke_with_conflict_check'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions/create_file.rb:60:in `invoke!'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions.rb:94:in `action'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions/create_file.rb:25:in `create_file'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/actions/file_manipulation.rb:115:in `template'
	from /opt/rubies/2.2.2/lib/ruby/gems/2.2.0/gems/ember-cli-rails-0.5.4/lib/generators/ember-cli/heroku/heroku_generator.rb:8:in `copy_package_json_file'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `block in invoke_all'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `each'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `map'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `invoke_all'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/group.rb:232:in `dispatch'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/railties-4.2.1/lib/rails/generators.rb:157:in `invoke'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/railties-4.2.1/lib/rails/commands/generate.rb:13:in `<top (required)>'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-4.2.1/lib/active_support/dependencies.rb:274:in `require'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-4.2.1/lib/active_support/dependencies.rb:274:in `block in require'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-4.2.1/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-4.2.1/lib/active_support/dependencies.rb:274:in `require'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/railties-4.2.1/lib/rails/commands/commands_tasks.rb:123:in `require_command!'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/railties-4.2.1/lib/rails/commands/commands_tasks.rb:130:in `generate_or_destroy'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/railties-4.2.1/lib/rails/commands/commands_tasks.rb:50:in `generate'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/railties-4.2.1/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/railties-4.2.1/lib/rails/commands.rb:17:in `<top (required)>'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-4.2.1/lib/active_support/dependencies.rb:274:in `require'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-4.2.1/lib/active_support/dependencies.rb:274:in `block in require'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-4.2.1/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-4.2.1/lib/active_support/dependencies.rb:274:in `require'
	from /Users/justinchapman/src/app/bin/rails:8:in `<top (required)>'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-4.2.1/lib/active_support/dependencies.rb:268:in `load'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-4.2.1/lib/active_support/dependencies.rb:268:in `block in load'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-4.2.1/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-4.2.1/lib/active_support/dependencies.rb:268:in `load'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from -e:1:in `<main>'
``

**Comment by seanpdoyle** [comment]
@thejchap thanks for opening this issue and bringing this to our attention!

This has been fixed in https://github.com/thoughtbot/ember-cli-rails/pull/296 and will be available in `0.5.5`.

**Comment by thejchap** [comment]
@seanpdoyle nice! thanks for the speedy reply and fix
